src/components/Admin/AdminBookings/AdminBooking/AdminBooking.tsx

```diff
@@ -31,17 +32,23 @@ const AdminBooking = ({ match }: any): JSX.Element => (
               </div>
               <div>
                 <p>
-                  Total Amount:
-                  {booking.guestTotalAmount ? numberToLocaleString(booking.guestTotalAmount, booking.currency) : '-'}
+                  Number of Guests: <span>{booking.numberOfGuests}</span>
+                </p>
+              </div>
+              <div>
+                <p>
+                  Total Amount:{' '}
+                  {booking.guestTotalAmount ? numberToLocaleString(booking.guestTotalAmount, booking.currency) : '-'}{' '}
                   <span>{booking.currency}</span>
                 </p>
               </div>
               <div>
                 <p>
-                  Listing: <BeeLink to={`/admin/listings/${booking.listingId}/edit`}>{booking.listingId}</BeeLink>
+                  Listing: <BeeLink to={`/admin/listings/${booking.listing.id}/edit`}>{booking.listing.id}</BeeLink>
                 </p>
               </div>
               <div>
+                <p>Status: {getAdminBookingDisplayStatus(booking.status)}</p>
```

Comment:
did amy want to see the guest info too? email, etc

Reply:
ya will add 🙏